Gamma_Analysis.py

```diff
@@ -63,8 +63,8 @@ def isotope_activity(isotope, emission_rates, emission_uncertainty):
     weight = []
     squares_total = []
     for i in range(len(branching_ratio)):
-        activity.append(emission_rates[i]/branching_ratio[i])
-        uncertainty.append(emission_uncertainty[i]/branching_ratio[i])
+        activity.append(emission_rates[i]/(.01*branching_ratio[i]))
```

Comment:
Could you fix the style in this section with spaces around operators? (except `**`)